Formula/httpstat.rb

```diff
@@ -0,0 +1,17 @@
+class Httpstat < Formula
+  desc "curl statistics made simple"
+  homepage "https://github.com/reorx/httpstat"
+  url "https://github.com/reorx/httpstat/archive/v1.1.2.tar.gz"
+  sha256 "0468abebfbae9ff7ea35d577b60ff9af9db3ff67ffd81b11ac005c4e9602919b"
+
+  bottle :unneeded
+
+  def install
+    bin.install "httpstat.py"
+    mv bin/"httpstat.py", bin/"httpstat"
+  end
+
+  test do
+    system "#{bin}/httpstat", "https://github.com"
```

Comment:
I think the output validation is not necessary, since I have covered all situation that may cause the program to fail, as long as it return 0, it means it can work properly. Even if github becomes 500 error doesn't got anything to do with httpstat's success or fail.